src/main/java/com/uid2/admin/audit/AuditHandler.java

```diff
@@ -1,6 +1,8 @@
 package com.uid2.admin.audit;
 
+import java.util.List;
+
 @FunctionalInterface
 public interface AuditHandler<E> {
-    OperationModel handle(E var);
+    List<OperationModel> handle(E var);
```

Comment:
This interface seems unnecessary. How about remove this and change AuditMiddleware interface to:
`Handler<RoutingContext> handle(Function<RoutingContext, List<OperationModel>> handler);`

Reply:
Didn't know Function was a thing; I did feel that including this interface made things awkward. Thank you for the solution